### ISSUE TITLE
SelectOneRadio : aria-checked unavailable in 14.0.10+

### ISSUE THREAD

**Issue by seelipraneeth** [primefaces/primefaces#13327]
after render, the translated html doesn't have the aria-checked attribute in div (for selectOneRadio) which existed till 14.0.9. Was it completely removed? It has been used in our application for selenuim test cases. how to assert now whether it has been check or not? Thank you

**Comment by melloware** [comment]
@seelipraneeth yes that was an incorrect attribute and throwing off the screenreaders which is the whole point of ARIA is helping the visually impaired.

See this ticket: https://github.com/primefaces/primefaces/issues/13047

**Comment by melloware** [comment]
actually it looks like I can put that back.  The ROLE="RADIO" is what was messing up the screenreader.  I will fix this for 14.0.12.

**Comment by seelipraneeth** [comment]
Thank you, Please let us know when can this version (14.0.12) be available ?
